packages/manager/modules/bm-server-components/src/netboot/template.html

```diff
@@ -0,0 +1,234 @@
+                            ></li>
+                            <li
+                                data-translate="server_configuration_netboot_title_help_rootdevice_list3"
+                            ></li>
+                            <li
+                                data-translate="server_configuration_netboot_title_help_rootdevice_list4"
+                            ></li>
+                            <li
+                                data-translate="server_configuration_netboot_title_help_rootdevice_list5"
+                            ></li>
+                            <li
+                                data-translate="server_configuration_netboot_title_help_rootdevice_list6"
+                            ></li>
+                            <li
+                                data-translate="server_configuration_netboot_title_help_rootdevice_list7"
+                            ></li>
+                        </ul>
+                    </oui-message>
+
+                    <div class="oui-list"
+                        data-ng-if="$ctrl.currentNetboot.network">
+                        <ul>
+                          <li class="oui-list__group">
+                            <strong>
+                                <span data-translate="server_configuration_netboot_kernel_option"></span>
+                            </strong>
+                            <ul>
+                                <li
+                                    data-ng-repeat="kerneloption in $ctrl.currentNetboot.network.options | filter:$ctrl.getActiveOptions track by $index"
+                                    data-ng-bind="kerneloption.option"
+                                ></li>
+                            </ul>
+                          </li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </oui-step-form>
+        <oui-step-form
+            data-header="{{:: 'server_configuration_netboot_review' | translate}}"
+            data-editable="!$ctrl.loading.setNetboot"
+            data-submit-text="{{ ::'server_configuration_netboot_confirm' | translate }}"
+            data-cancel-text="{{ :: 'server_configuration_netboot_cancel' | translate }}"
+            data-on-cancel="$ctrl.goBack()"
+        >
+            <p
+                data-ng-bind="'server_configuration_netboot_step2_question' | translate:{ t0: $ctrl.server.name }"
+            ></p>
+            <ul class="list-unstyled">
+                <li>
+                    <strong
+                        data-ng-bind=":: 'server_configuration_netboot_step2_type' | translate"
+                    ></strong>
+                    <span
+                        data-translate="{{ 'server_configuration_netboot_option_' + $ctrl.currentNetboot.type }}"
+                    ></span>
+                </li>
+                <li data-ng-if="$ctrl.currentNetboot.type === $ctrl.RESCUE">
```

Comment:
Can you set these tests in specific methods ? (All of the `.type === $ctrl.TYPE`)